plugins/cilium-cni/chaining/generic-veth/generic-veth.go

```diff
@@ -49,11 +51,17 @@ func (f *GenericVethChainer) Add(ctx context.Context, pluginCtx chainingapi.Plug
 	}()
 	var (
 		hostMac, vethHostName, vethLXCMac, vethIP, vethIPv6 string
+		routes                                              []netlink.Route
 		vethHostIdx, peerIndex                              int
 		peer                                                netlink.Link
 		netNs                                               ns.NetNS
 	)
 
+	conf, err := ciliumcniutils.GetConfigFromCiliumAgent(cli)
```

Comment:
Instead of factoring this out into `ciliumcniutils` and calling the Cilium agent OpenAPI call a second time to retrieve the agent config here, how about storing the result of the initial call in `cmdAdd` as `*models.DaemonConfigurationStatus` in `PluginContext` and then use it here? The agent config is not expected to change in between the calls.

Reply:
Make sense.  I have updated accordingly.